### DISCUSSION TITLE
搞不清楚在講哪一種資料型態

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#160] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/86505454/142744854-34a48a84-f06f-4766-8928-a3a6c8b35e32.png)
我想問的是：text(strings)是一種怎麼樣的資料型態？什麼資料型態叫作text(strings)？謝謝。

**Comment by enoch-lightning** [comment]
🔴 🧧 🟥 🚗 🐷 🐖 🐽 🚡 🚂 🚇 🚋 🚀 🚀 🦝

**Comment by Jie2222** [comment]
應該沒有這種資料型態吧:sweat_smile::sweat_smile:

**Comment by EricAgumonJou** [comment, marked as answer]
我只知道有字串型態：string （也就是text文字）
老師是希望我們自己練習看看用string（或稱為text）是可以運用成員運算子（in）。

**Comment by EricAgumonJou** [comment]
string(或稱text) 使用in成員運算子的範例：
<img width="466" alt="Screen Shot 2021-11-24 at 15 14 11" src="https://user-images.githubusercontent.com/91121500/143192333-8ab1c71b-76f2-4778-9d08-89ee58afd66d.png">
：
